src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php

```diff
@@ -232,9 +233,17 @@ protected function GetMLTFilterQuery()
             $oTableList->sRestriction = null; // do not include the restriction - it is part of the parent table, not the mlt!
 
             $sFilterQuery = $oTableList->FilterQuery().$this->GetMLTRecordRestrictions();
-            $sFilterQueryOrderInfo = $oTableList->GetSortInfoAsString();
-            if (!empty($sFilterQueryOrderInfo)) {
-                $sFilterQuery .= ' ORDER BY '.$sFilterQueryOrderInfo;
+//            $sFilterQueryOrderInfo = $oTableList->GetSortInfoAsString();
```

Comment:
We can't remove this. This is needed to allow custom sort orders.

Reply:
![Selection_236](https://user-images.githubusercontent.com/20949224/89501495-bb592e80-d7c3-11ea-9cf3-e4ebcabb7440.png)

This sorting (the one of cms_tbl_conf: "position ASC") is the main reason why it currently looks broken.
It defies the check in the rendering code: if ($currentGroup !== $activeGroup)
As there is no group (tab) ordering in the data then.